### ISSUE TITLE
Error "ERR_BLOCKED_BY_RESPONSE" for Brave Official Website and ProtonMail Ad Partner Sites Access Blocked

### ISSUE THREAD

**Issue by Kosezu** [brave/brave-browser#33599]
## Description 
[Just Updated Brave Beta to Latest Build and Getting Blocked Error for Brave Official Website Along with ProtonMail](https://community.brave.com/t/just-updated-brave-beta-to-latest-build-and-getting-blocked-error-for-brave-official-website-along-with-protonmail/510016)

Title. Visiting [Brave.com](http://brave.com/) I get the following error, after updating to latest Brave beta build (Version 1.60.86 Chromium: 118.0.5993.54):
brave.com is blocked
brave.com refused to connect.
ERR_BLOCKED_BY_RESPONSE

Same response for [mail.proton.me](http://mail.proton.me/), just replace [brave.com](http://brave.com/) with the proton address and it’s the same exact error.

Any and all [brave.com/(...)](http://brave.com/(...)) addresses
For Proton:
[proton.me](http://proton.me/)
[mail.proton.me](http://mail.proton.me/)
[account.proton.me/login](http://account.proton.me/login)

HOWEVER [account.protonvpn.com](http://account.protonvpn.com/) DOES WORK

## Steps to Reproduce
   1. Error occurs under my Brave Profile and Private Window under my profile.
   2. Does not occur under a new profile.
 

## Actual result:
Get error:
brave.com is blocked
brave.com refused to connect.
ERR_BLOCKED_BY_RESPONSE

and 

mail.proton.me is blocked
mail.proton.me refused to connect.
ERR_BLOCKED_BY_RESPONSE

## Expected result:
Expect to visit site normally

## Reproduces how often: 
Happens every time, under my profile

## Brave version (brave://version info)

Brave | 1.60.88 Chromium: 118.0.5993.70 (Official Build) beta (x86_64)
-- | --
Revision | ff9150ac5dd9934a7f431ddf478ad3e45ae68546
OS | macOS Version 14.0 (Build 23A344)

## Version/Channel Information:
Does this issue happen on any other channels? Or is it specific to a certain channel?
NO, specific to beta channel, but only my profile. It seems something was broken during the update.

- Can you reproduce this issue with the current release? No
- Can you reproduce this issue with the beta channel? No, I run beta on a Windows computer as well, and same version works there.
- Can you reproduce this issue with the nightly channel? No

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NO
- Does the issue resolve itself when disabling Brave Rewards? NO
- Is the issue reproducible on the latest version of Chrome? NO

**Comment by Kosezu** [comment]
Similar issue as this [ERR_BLOCKED_BY_RESPONSE when navigating to Google.com #30162]( https://github.com/brave/brave-browser/issues/30162)

**Comment by hiro5id** [comment]
I have the exact same problem..

Brave version info:

Brave	1.60.114 Chromium: 119.0.6045.124 (Official Build) (x86_64) 
Revision	99a34933fc6a44fdfe52dea41341927bbce3549a
OS	macOS Version 14.1.1 (Build 23B81)
JavaScript	V8 11.9.169.6

**Comment by hmijail** [comment]
Same problem. 
Version 1.60.118 Chromium: 119.0.6045.163 (Official Build) (arm64)
on macOS 13.6.1

I rarely use Brave, so opening it and seeing its own website page seemingly blocked by itself made me think at first that this was some marketing stunt to show off the onion redirect (which does load the page).

**Comment by eylles** [comment]
yep it still works on 1.61.100 didn't happen back in 1.59.117, both on debian sid

![image](https://github.com/brave/brave-browser/assets/33670677/39ecd254-1ffc-4846-9369-af3a17e87275)

|Brave|	1.61.100 Chromium: 120.0.6099.62 (Official Build) (64-bit) |
|----|----|
|Revision|	63d361b4e5fd163e032192aa8e033cf85f846575|
|OS|	Linux|

**Comment by ok412** [comment]
Can confirm that disabling "Automatically redirect .onion sites" under brave://settings/privacy fixes this. 

Brave details; 1.61.114 on Windows 10

**Comment by cheecheeo** [comment]
> Can confirm that disabling "Automatically redirect .onion sites" under brave://settings/privacy fixes this.
> 
> Brave details; 1.61.114 on Windows 10

this also worked for me.

**Comment by bsclifton** [comment]
cc: @darkdh @fmarier on the `Automatically redirect .onion sites` issue ^^

**Comment by BrunelleschiGH** [comment]
Just encountered this on version 1.67.134 when attempting to navigate to Google websites (Google.com & youtube.com tested).

Disabling `Only resolve .onion addresses in Tor windows` appears to still be a valid workaround for this. Issue still needs resolution (if it hasn't been fixed in another Issue here).

**Comment by fmarier** [comment]
@BrunelleschiGH That's a different work-around than the one mentioned in https://github.com/brave/brave-browser/issues/33599#issuecomment-1882421824. We have removed the `Automatically redirect .onion sites` option in https://github.com/brave/brave-browser/issues/36933 and so the original issue should be fixed. Also `google.com` and `youtube.com` don't have `.onion` sites and so they were never affected by the now-removed setting.

Could you file a separate issue so that we can investigate the problem you're seeing?

**Comment by fmarier** [comment]
Marking as a "duplicate" of https://github.com/brave/brave-browser/issues/36933.
